js-foundations.js

```diff
@@ -2,26 +2,53 @@
 // Coding practical questions
 
 // 1. Write an arrow function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
+var num = 15
+var num2 = 0
+var num3 = 385
+var num4 = -7
 
+let numChecker = (number) =>{
+    if(number%3 ===0){
+     return number + " is a divisible by 3"
+     }else
+        return number + " is not divisible by 3"
+}
 
+console.log(numChecker(0))
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
+var helloMe = {
+    firstName: "Jonathon",
+    lastName: "Arbona",
+    height: "6'2",
+    eyes: "blue",
+    getData: function() {return this.firstName + " " + this.lastName + " is " + this.height + " and has " + this.eyes + " eyes."}
```

Comment:
This is great, to make it even more simple consider using _Template literals (Template strings)_.